### DISCUSSION TITLE
Premium Showcase/Feature comparison

### DISCUSSION THREAD

**Discussion by dukemagus** [AssetRipper/AssetRipper#1185] [Ideas]
Can you make a video or a post with some screenshots comparing the output and tools of the premium and the open source version in a project? 

 Showing the side by side result on a real case (decompiling a real game) could help visualize the different results and how much work is saved or increased depending on the choice.

**Comment by ds5678** [comment]
I'm a little strapped for time right now, but I definitely want to make videos like one you described.
